client/src/components/Comics.js

```diff
@@ -7,6 +7,10 @@ import Loading from "./Loading";
 
 function Comics() {
   const [comics, setComics] = useState([]);
+  const [pageSize, setPageSize] = useState(6);
+  const [pageMin, setPageMin] = useState(6);
+  const [pageMax, setPageMax] = useState(36);
+  const [pageStep, setPageStep] = useState(6);
```

Comment:
Sorry, when I said to extract these to constants I meant something like
```
const PER_PAGE_MIN = 6
const PER_PAGE_MAX = 36
const PER_PAGE_STEP = 6
```
These shouldn't be stored in state unless we want to modify them